releases/2020.08.yaml

```diff
@@ -0,0 +1,133 @@
+repositories:
+  qpOASES:
+    type: git
+    url: https://github.com/robotology-dependencies/qpOASES.git
+    version: v3.2.0.1
+  osqp:
+    type: git
+    url: https://github.com/oxfordcontrol/osqp.git
+    version: v0.6.0
+  ycm:
+    type: git
+    url: https://github.com/robotology/ycm.git
+    version: v0.11.1
+  YARP:
+    type: git
+    url: https://github.com/robotology/yarp.git
+    version: master
+  ICUB:
+    type: git
+    url: https://github.com/robotology/icub-main.git
+    version: devel
+  ICUBcontrib:
+    type: git
+    url: https://github.com/robotology/icub-contrib-common.git
+    version: v1.16.0
+  robots-configuration:
+    type: git
+    url: https://github.com/robotology/robots-configuration.git
+    version: v1.16.0
+  GazeboYARPPlugins:
+    type: git
+    url: https://github.com/robotology/gazebo-yarp-plugins.git
+    version: devel
+  icub-gazebo:
+    type: git
+    url: https://github.com/robotology/icub-gazebo.git
+    version: v1.16.0
+  yarp-matlab-bindings:
+    type: git
+    url: https://github.com/robotology/yarp-matlab-bindings.git
+    version: master
+  RobotTestingFramework:
+    type: git
+    url: https://github.com/robotology/robot-testing-framework.git
+    version: v2.0.1
+  icub-tests:
+    type: git
+    url: https://github.com/robotology/icub-tests.git
+    version: v1.16.0
+  blockTest:
+    type: git
+    url: https://github.com/robotology/blocktest.git
+    version: d490bd018cf6c40e0a2e37650943d43de23751a0
+  blocktest-yarp-plugins:
+    type: git
+    url: https://github.com/robotology/blocktest-yarp-plugins.git
+    version: 78eacbf220a1c6db6d1bac68e4b51ab4a3a55142
```

Comment:
It wasn't sufficient writing `master`?

Reply:
Ideally the release file should never contain branches, but always fixed tags or commits to ensure that we are always ready to do a release. As a temporary workaround we have branches for yarp/icub due to the fact that 3.4 and 1.16 should happen before august, but given that for blockTest I have no idea of when the release will be done, I prefer to played safe.